### PR TITLE
api/gadgettracermanager: Support new protoc-gen-go-grpc

### PR DESCRIPTION
**Pull request by blanquicet**
# Support new protoc-gen-go-grpc

As described https://github.com/golang/protobuf/issues/1070, plugins=grpc is not supported anymore in latest versions of protoc-gen-go because it was moved to https://github.com/grpc/grpc-go project, check [this](https://github.com/golang/protobuf/issues/903).

Therefore, this PR updates api/gadgettracermanager to use the new syntax and regenerates the code.

Consider that this PR was created in preparation for a coming PR where I need to modify the gadgettracermanager API.

**TO-DO**:
- [x] With changes introduced in https://github.com/grpc/grpc-go/pull/3657, the service implementations must embed the corresponding Unimplemented<ServiceName>Server for future compatibility. I am not sure how to do that so I set option require_unimplemented_servers to false to enable backward compatibility without requiring code changes. (Here the full discussion about this topic by grpc community: https://github.com/grpc/grpc-go/issues/3669).

### CODE REVIEW

**Comment by blanquicet** [comment]
> This PR removes an import of `github.com/golang/protobuf` in the generated code. You can run `go mod tidy` to update go.mod accordingly.

Done

File: pkg/gadgettracermanager/gadgettracermanager.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ import (
 import "C"
 
 type GadgetTracerManager struct {
+	pb.UnimplementedGadgetTracerManagerServer
```
Comment (alban) [inline pkg/gadgettracermanager/gadgettracermanager.go:41]: Is it necessary to add this here? It seems to provide default implementation that `GadgetTracerManager` will override anyway. I'm not saying to remove it, just trying to understand.

**Review by alban — APPROVED** [review]
I have one question, but LGTM.

File: pkg/gadgettracermanager/gadgettracermanager.go (hunk 1 above)
Comment (blanquicet) [reply pkg/gadgettracermanager/gadgettracermanager.go:41]: AFAIU, to register services using the methods generated by `protoc-gen-go-grpc`, we **must** do this or disable this requirement by using option require_unimplemented_servers without modifying the code. However, the second option is not recommended: https://github.com/grpc/grpc-go/blob/master/cmd/protoc-gen-go-grpc/README.md#future-proofing-services

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM. I think it fixes https://github.com/kinvolk/inspektor-gadget/issues/147

**Pull request merged**